packages/@react-aria/table/docs/useTable.mdx

```diff
@@ -488,12 +492,823 @@ function Checkbox(props) {
+                  widths={widths}
+                  /*- end highlight -*/
+                />
+              )
+            )}
+          </ResizableTableRow>
+        ))}
+      </ResizableTableRowGroup>
+    </table>
+  );
+}
+```
+
+</details>
+
+
+### Resizable table header
+
+The `TableRowGroup` and `TableHeaderRow` only require minor style changes to accommodate the the new `display` changes made in
+`ResizableColumnsTable`.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+function ResizableTableRowGroup({type: Element, style, children}) {
+  let {rowGroupProps} = useTableRowGroup();
+  return (
+    <Element
+      {...rowGroupProps}
+      style={{
+        /*- begin highlight -*/
+        display: 'block',
+        /*- end highlight -*/
+        ...style
+      }}>
+      {children}
+    </Element>
+  );
+}
+```
+
+</details>
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+function ResizableTableHeaderRow({item, state, children}) {
+  let ref = useRef();
+  let {rowProps} = useTableHeaderRow({node: item}, state, ref);
+
+  return (
+    <tr
+      {...rowProps}
+      ref={ref}
+      /*- begin highlight -*/
+      style={{display: 'flex'}}
+      /*- end highlight -*/
+    >
+      {children}
+    </tr>
+  );
+}
+```
+
+</details>
+
+The `TableColumnHeader` is where we see the bulk of the changes required to support resizable columns. First of all, we need to accommodate a `Resizer` element in every resizable column that
+the user can drag or focus to perform a resize operation. To avoid interfering with the existing grid keyboard navigation, we only display this resizer when
+the column is hovered or if the column itself is being resized. This however introduces another issue: how will keyboard, screen reader, and touch users begin a resize operation if the resizer
+is only visible on hover? We can't just have the user press on the column header since that action is reserved for sorting.
+
+To resolve this, we need to render a menu button as the column header if the column is resizable, allowing non-mouse users to trigger any available action on the column. For resizing, this would
+cause focus to shift to the resizer itself, displaying the resizer for touch screen users to drag and letting keyboard and screen readers resize the column via keyboard arrows and screenreader operations
+accordingly.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+// Reuse the MenuButton from your component library. See below for details.
+import {MenuButton} from 'your-component-library';
+import {useHover} from '@react-aria/interactions';
+
+function ResizableTableColumnHeader({column, state, layoutState, onResizeStart, onResize, onResizeEnd}) {
```

Comment:
This is quite the change to the aria example's TableColumnHeader but IMO is good to have to keep interaction handling for all interaction types. Could be reduced if we wanna omit sorting handling but IMO its good to keep the previous functionality from the original aria example

Reply:
Sounds good, I'll go ahead and make the docs example simple and do a tailwind example with all the bells and whistles